component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -849,7 +849,7 @@ public List<Property> getConfigurationProperties() {
         smsUrl.setDisplayName("SMS URL");
         smsUrl.setRequired(true);
         smsUrl.setDescription("Enter client sms url value. If the phone number and text message are in URL, " +
-                "specify them as $ctx.num and $ctx.msg");
+                "specify them as $ctx.num and $ctx.msg or $ctx.otp");
```

Comment:
Shall we change the description as follows to avoid any confusion?

```
smsUrl.setDescription("Enter client sms url value. If the phone number, text message, and OTP are in URL, " +
                "specify them as $ctx.num, $ctx.msg, and $ctx.otp");
```